packages/api-reference/src/components/Content/Schema/Schema.vue

```diff
@@ -1,23 +1,23 @@
 <script lang="ts" setup>
 import { Disclosure, DisclosureButton, DisclosurePanel } from '@headlessui/vue'
 import { ScalarIcon, ScalarMarkdown } from '@scalar/components'
-import type { OpenAPIV3_1 } from '@scalar/openapi-types'
-import { computed, ref, watch } from 'vue'
+import type { OpenAPIV2, OpenAPIV3, OpenAPIV3_1 } from '@scalar/openapi-types'
+import { computed, inject } from 'vue'
 
-import {
-  getDiscriminatorMapping,
-  getDiscriminatorPropertyName,
-  mergeDiscriminatorSchemas,
-} from '@/components/Content/Schema/helpers/schema-discriminator'
-import SchemaDiscriminator from '@/components/Content/Schema/SchemaDiscriminator.vue'
 import ScreenReader from '@/components/ScreenReader.vue'
+import type { Schemas } from '@/features/Operation/types/schemas'
+import { DISCRIMINATOR_CONTEXT } from '@/hooks/useDiscriminator'
 
 import SchemaHeading from './SchemaHeading.vue'
 import SchemaProperty from './SchemaProperty.vue'
 
 const props = withDefaults(
   defineProps<{
     value?:
+      | OpenAPIV2.DefinitionsObject
```

Comment:
@antlio we are upgrading now so we only need V3_1 everywhere. We should actually alias it somewhere as the current OpenAPI version in references then use that everywhere. I will do it in my sidebar PR 😄

Reply:
lgmt!